.travis.yml

```diff
@@ -139,19 +139,28 @@ jobs:
       # Workaround for bug in libopus's opus.h including <opus_multistream.h>
       # instead of <opus/opus_multistream.h>.
       # Virtual X (Xvfb) is needed for analyzer waveform tests
+      addons:
+        homebrew:
+          update: true
+          packages:
+            - ccache
       env: >-
-        CMAKEFLAGS_EXTRA="-DLOCALECOMPARE=OFF -DCMAKE_PREFIX_PATH=/usr/local/opt/qt5/"
+        MIXXX_ENVNAME=2.3-j00004-497fe02e-osx10.11-x86_64-release
```

Comment:
Can you get Travis to read the build/osx/golden_environment file? If we have to update that in multiple places, we will forget to update it everywhere.

Reply:
What file? I can't find it... https://downloads.mixxx.org/builds/buildserver/2.3.x-macosx/

Reply:
It's in this Git repository.